docs/content/operations/dashboard.md

```diff
@@ -3,24 +3,32 @@
 See What's Going On
 {: .subtitle }
 
-The dashboard is the central place that shows you the current active routes handled by Traefik. 
+The dashboard is the central place that shows you the current active routes handled by Traefik.
 
 <figure>
     <img src="../../assets/img/webui-dashboard.png" alt="Dashboard - Providers" />
     <figcaption>The dashboard in action</figcaption>
 </figure>
 
-By default, the dashboard is available on `/dashboard` on port `:8080`.
-There is also a redirect of `/` to `/dashboard`, but one should not rely on that property as it is bound to change,
-and it might make for confusing routing rules anyway.
+The dashboard is available at the same location as the [API](./api.md) but on the path `/dashboard/` by default.
 
-!!! info "Did You Know?"
-    It is possible to customize the dashboard endpoint. 
-    To learn how, refer to the [API documentation](./api.md)
-    
-## Enabling the Dashboard
+!!! warning "The trailing slash `/` in `/dashboard/` is mandatory"
 
-To enable the dashboard, you need to enable [Traefik's API](./api.md).
+There are 2 ways to configure and access the dashboard:
+
+- [Secured mode (Recommended)](#secured-mode)
+- [Insecured mode](#insecured-mode)
```

Comment:
```suggestion
- [Insecure mode](#insecure-mode)
```

Reply:
insecured does not seem to be an actual word. which means you'll have fix the anchors too, sorry.